public/pages/CreateMonitor/components/VisualGraph/utils/helpers.js

```diff
@@ -226,3 +192,12 @@ export function getAggregationGraphHint(hint) {
     ')'
   );
 }
+
+export function getGraphDescription(bucketValue, bucketUnitOfTime, groupBy) {
+  const text = `FOR THE LAST ${bucketValue}  ${selectOptionValueToText(
```

Comment:
NP: Looks like there are two spaces between `${bucketValue}` and `${selectOptionValueToText()}`

Reply:
Fixed